<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RSA_DEClass</class>
 <widget class="QMainWindow" name="RSA_DEClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1153</width>
    <height>672</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RSA_DE</string>
  </property>
  <property name="styleSheet">
   <string notr="true">border-top-color: rgb(170, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="styleSheet">
    <string notr="true">#centralWidget{border-image: url(:/RSA_DE/Resources/images/nanyou.jpg);}

</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="Btn_GenKey_slt">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>35</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgba(255, 255, 255, 100);
font: 9pt &quot;幼圆&quot;;</string>
        </property>
        <property name="text">
         <string>生成密钥</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Btn_ED_slt">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>35</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgba(255, 255, 255, 100);
font: 9pt &quot;幼圆&quot;;</string>
        </property>
        <property name="text">
         <string>加解密</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>378</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="page">
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>118</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="Btn_Open_File">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>35</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgba(255, 255, 255, 100);
font: 9pt &quot;幼圆&quot;;</string>
            </property>
            <property name="text">
             <string>打开文件</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_3">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>138</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="Btn_Save_file">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>35</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgba(255, 255, 255, 100);
font: 9pt &quot;幼圆&quot;;</string>
            </property>
            <property name="text">
             <string>保存文件</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_4">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>158</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <widget class="QPlainTextEdit" name="plain_PlainText">
            <property name="styleSheet">
             <string notr="true">background-color: rgba(255, 255, 255, 100);
font: 9pt &quot;Consolas&quot;;</string>
            </property>
            <property name="plainText">
             <string>请输入明文或者密文...</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>88</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QComboBox" name="com_Select">
              <property name="minimumSize">
               <size>
                <width>80</width>
                <height>32</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>100</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color: rgba(255, 255, 255, 100);
font: 9pt &quot;幼圆&quot;;</string>
              </property>
              <item>
               <property name="text">
                <string>加密</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>解密</string>
               </property>
              </item>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="Btn_StartED">
              <property name="maximumSize">
               <size>
                <width>80</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color: rgba(255, 255, 255, 100);
font: 9pt &quot;幼圆&quot;;</string>
              </property>
              <property name="text">
               <string>开始</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_3">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>98</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QPlainTextEdit" name="plain_Ciper">
            <property name="styleSheet">
             <string notr="true">background-color: rgba(255, 255, 255, 100);
font: 9pt &quot;Consolas&quot;;</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_2">
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLineEdit" name="Line_KeySize">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>32</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>140</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgba(255, 255, 255, 100);
font: 9pt &quot;Consolas&quot;;</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="placeholderText">
             <string>请输入密钥位数</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="Btn_Gen">
            <property name="styleSheet">
             <string notr="true">background-color: rgba(255, 255, 255, 100);
font: 9pt &quot;幼圆&quot;;</string>
            </property>
            <property name="text">
             <string>开始生成</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>458</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QLabel" name="label_Pub_E">
              <property name="maximumSize">
               <size>
                <width>50</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>公钥E</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPlainTextEdit" name="plain_Pub_E">
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>100</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color: rgba(255, 255, 255, 100);
font: 9pt &quot;Consolas&quot;;</string>
              </property>
              <property name="lineWidth">
               <number>1</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_Pub_N">
              <property name="maximumSize">
               <size>
                <width>50</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>公钥N</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPlainTextEdit" name="plain_Pub_N">
              <property name="styleSheet">
               <string notr="true">background-color: rgba(255, 255, 255, 100);
font: 9pt &quot;Consolas&quot;;</string>
              </property>
              <property name="lineWidth">
               <number>10</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QLabel" name="label_Pri_D">
              <property name="maximumSize">
               <size>
                <width>50</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>私钥D</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPlainTextEdit" name="plain_Pri_D">
              <property name="styleSheet">
               <string notr="true">background-color: rgba(255, 255, 255, 100);
font: 9pt &quot;Consolas&quot;;</string>
              </property>
              <property name="lineWidth">
               <number>6</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1153</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="styleSheet">
    <string notr="true">border-top-color: rgb(0, 0, 0);</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../RSA_DE.qrc"/>
 </resources>
 <connections/>
</ui>
